src/main/java/com/incentives/piggyback/user/controller/UserController.java

```diff
@@ -63,6 +88,15 @@
         log.debug("User Service: Received DELETE request for deleting user with userid."+ id);
         if(userService.findById(id).isPresent()) {
              userService.deleteById(id);
+            //PUSHING MESSAGES TO GCP
+            messagingGateway.sendToPubsub(
+                    CommonUtility.stringifyEventForPublish(
+                            id.toString(),
+                            Constant.USER_DEACTIVATED_EVENT,
+                            Calendar.getInstance().getTime().toString(),
+                            HttpStatus.OK.toString(),
```

Comment:
4th event field is meant for specifying partnerId, so that an event can be mapped to a partner - for billing and invoicing. For events not related to partners directly, it should be passed as empty.